embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/utils/NetworkUtils.kt

```diff
@@ -74,11 +74,7 @@ internal object NetworkUtils {
      * @return the URL with the hash fragment and query string parameters removed
      */
     @JvmStatic
-    fun stripUrl(url: String?): String? {
-        if (url == null) {
-            return null
-        }
-
+    fun stripUrl(url: String): String {
```

Comment:
Can move this. method to the logging service? It's only called there and in the test, and the test could just validate the URL is stripped without using that method to get the expected match?